.github/workflows/model-monitoring-gsq-ci.yml

```diff
@@ -0,0 +1,105 @@
+name: model-monitoring-gsq-ci
+
+on:
+  pull_request:
+    branches:
+      - main 
+    paths:
+      - assets/model_monitoring/**
+      - .github/workflows/model-monitoring-gsq-ci.yml
+      
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+env:
+  momoComponentsRootPath: assets/model_monitoring/components
+  testsRootPath: assets/model_monitoring/components/tests
+  pytest_report_folder: results
+  pytest_report_file: junit3.xml
+  scripts_setup_dir: scripts/setup
+permissions:
+  # Required to clone repo
+  contents: read
+  # Required for OIDC login to Azure
+  id-token: write
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  check-execution-context:
+    uses: Azure/azureml-assets/.github/workflows/check-execution-context.yaml@main
+  run-momo-tests:
+    name: Run GSQ Monitoring Component Tests
+    runs-on: ubuntu-latest
+    needs: check-execution-context
+    environment: Testing
+    steps:
+      - name: Clone branch
+        uses: Azure/azureml-assets/.github/actions/clone-repo@main
+        with:
+          forked-pr: ${{ needs.check-execution-context.outputs.forked_pr }}
+      - name: Use Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+
+      - name: Install fusepy
+        run: sudo apt-get install -y fuse
+
+      - name: Log in to Azure and create resources
+        uses: ./.github/actions/create-azure-resources
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+          scripts-setup-dir: ${{ env.scripts_setup_dir }}
+
+      - name: Install dependencies
+        run: pip install -r ${{ env.testsRootPath }}/requirements.txt
+
+      - name: Upgrade pandas, numpy, numba
+        run: pip install pandas==2.0.3 numpy==1.24.4 numba==0.57.1
```

Comment:
why don't you have this in requirements.txt

Reply:
done, moved to separate file gsq-requirements.txt (the requirements are actually very different between the two set of components)